submissions/nauvian-perspective-bentham/nauvian-perspective-bentham.md

```diff
@@ -0,0 +1,50 @@
+*Something about Bentham voice and Xisuma text interview and such here*
```

Comment:
@Buckminsterfullerene02  Do you want to add something here? Or is this intended for the little intro we always have? You could, for example, say that this is a continuation of the Nauvian Perspective, and then you ended up doing two interviews in one. 

Although, it might be easier if this is simply included in our intro. So if it's fine with you we will cut this out.

Reply:
I'll write a little intro for the former. I put that in because I thought there was a process behind that done by someone else. But I feel like I should explicitly state the media type of both interviews and that the article is a continuation of your Nauvian Perspective series. I know that probably makes formatting a bit harder for you but hopefully it's okay if I put that in.

Reply:
Oh, I could have been clearer. Therenas always writes the main intro and outro. He will specify that it's a continuation of the series, and that you actually chatted with Bentham  and all those things. You're welcome to write your own if you want. Maybe he'll keep it or use it as inspiration, not sure. But having two intros would be too much I think.

Reply:
Yeah, I just sat down to write it and thought "eh, I think the intro Therenas writes is probably only required here".